pkg/txn/storage/tae/storage_debug.go

```diff
@@ -30,6 +29,13 @@ func (s *taeStorage) Debug(ctx context.Context,
 	switch opCode {
 	case uint32(debug.CmdMethod_Ping):
 		return s.handlePing(data), nil
+	case uint32(debug.CmdMethod_Flush):
+		_, err := handleRead(
+			ctx, s,
+			txnMeta, data,
+			s.taeHandler.HandleFlushTable,
+		)
+		return nil, err
```

Comment:
returns some string to indicate that the operation was performed successfully. e.g. "OK"